maas-schemas/schemas/maas-backend/subscriptions/subscriptions-intents-update-state/request.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/subscriptions/subscriptions-intents-update-state/request.json",
```

Comment:
Convert this to `subscriptions-intents-update` to be more general.

Reply:
The service actually uses in update state, so maybe we could go through backend changes first then decide if we keep this or not